files/scripts/swss.sh

```diff
@@ -3,6 +3,15 @@
 DEPENDENT="radv dhcp_relay"
 MULTI_INST_DEPENDENT="teamd"
 
+# Update dependent list based on other packages requirements
```

Comment:
Why only swss.sh?

Reply:
Needed by dhcp-relay. Could be in any other script as well on demand.

Reply:
Actually, the right way is to have a dependency script autogenerated, for others (bgp, teamd, frr) it means once they become packages the will automatically have it.